### DISCUSSION TITLE
Install MPD

### DISCUSSION THREAD

**Discussion by entuccio** [Homebrew/discussions#796] [Everyday usage]
Hello,

I have an old MAc Mini 2011 with Mac Os Sierra 10.12.6

If I try to install mpd with homebrew it says:

mpd: macOS Mojave or newer is required for this software.

I tried to install an old version with

brew install mpd@otherversion

but it doesn't work.

Any suggestion?

Thank you

**Comment by ugultopu** [comment]
Try

    brew tap-new entuccio/sierra
    brew extract --version=0.21 mpd entuccio/sierra
    brew install mpd@0.21

Let me know if it works or not.
